tests/integration/test_types_integration.py

```diff
@@ -212,69 +212,74 @@ def test_time(trino_connection):
         .add_field(
             sql="TIME '23:59:59.999'",
             python=time(23, 59, 59, 999000))
-        # TODO: re-enable when Trino 404 is released
-        # .add_field(
-        #     sql="TIME '23:59:59.9999'",
-        #     python=time(0, 0, 0))  # PARAMETRIC_DATETIME not enabled
-        # .add_field(
-        #     sql="TIME '23:59:59.99999'",
-        #     python=time(0, 0, 0))  # PARAMETRIC_DATETIME not enabled
-        # .add_field(
-        #     sql="TIME '23:59:59.999999'",
-        #     python=time(0, 0, 0))  # PARAMETRIC_DATETIME not enabled
-        # round down
         .add_field(
-            sql="TIME '00:00:00.000100'",
-            python=time(0, 0, 0))
+            sql="TIME '23:59:59.9999'",
+            python=time(23, 59, 59, 999900))
         .add_field(
-            sql="TIME '00:00:00.0000001'",
-            python=time(0, 0, 0))  # PARAMETRIC_DATETIME not enabled
```

Comment:
where did this go?

Reply:
Isn't this moved a few lines below (233)?

Reply:
Yeah, the diff on GitHub looked a bit wacky. I've re-reviewed locally in IDE and everything looks good.